core/src/services/s3/docs.md

```diff
@@ -103,6 +103,10 @@ async fn main() -> Result<()> {
     // - "http://127.0.0.1:9000"
     // - "https://oss-ap-northeast-1.aliyuncs.com"
     // - "https://cos.ap-seoul.myqcloud.com"
+    // - "https://storage.googleapis.com/" 
+    //    [Note]: To access GCS via S3 API, 
```

Comment:
Please add a new section called GCS XML API in compatible services docs instead of here.